GameStore.DataAccess/EntitiesConfigurations/GenreConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GameStore.DataAccess.EntityModels;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameStore.DataAccess.EntitiesConfigurations
+{
+    static class GenreConfiguration
+    {
+        public static void SetGenreConfiguration(DbModelBuilder modelBuilder)
+        {
+            if (modelBuilder == null)
```

Comment:
You have to signalize about wrong execution of your code. 
According to this code, it will be executed with no errors etc. and you will have wrong db schema.

Reply:
I removed checks on null. If it necessary i can add throwing exception.